mart/configs/model/detection/preprocessor_single_modal.yaml

```diff
@@ -0,0 +1,6 @@
+# @package model.modules.preprocessor
```

Comment:
Why not create a modules directory? Why does this live under detection when it has nothing to do with detection?

I also think this should just be `preprocessor.yaml` or something like that. Perhaps `8bit_preprocessor.yaml` or something indicating that this is doing 0-255 normalization?

Reply:
I named it `tuple_normalizer.yaml`